### ISSUE TITLE
Delta source_format key error

### ISSUE THREAD

**Issue by JoeBarcus** [databrickslabs/dlt-meta#52]
When using the delta option for the bronze source_format the read_dlt_delta method will fail with a key error:
```
.{bronze_dataflow_spec.sourceDetails["table"]}
KeyError: 'table'
```
Switching the key to `source_table` worked for me locally.

**Comment by ravi-databricks** [comment]
@JoeBarcus created [PR](https://github.com/databrickslabs/dlt-meta/pull/54) please check.

**Comment by ravi-databricks** [comment]
fixed in v0.0.8
